glossary/ePO_combined_glossary.html

```diff
@@ -2553,7 +2553,7 @@ <h2 xmlns:f="http://https://github.com/costezki/model2owl#">Attributes (datatype
                         graphical representation, or a free-text account of the resource.
                         
                         WG Approval 30/05/2023</td>
-                     <td>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [1..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br></td>
+                     <td>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [1..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..*]<br>rdf:PlainLiteral [0..1]<br>rdf:PlainLiteral [0..1]<br></td>
```

Comment:
These are changes unrelated to the hotfix issue, they are fixing new things. 

Normally, one should keep the changes atomic

Reply:
These changes are automatically generated by model2owl based on the eap binary file.